lib/mayaUsd/ufe/private/UfeNotifGuard.h

```diff
@@ -24,20 +25,42 @@ namespace ufe {
 class InPathChange
 {
 public:
-	InPathChange() { fInPathChange = true; }
-	~InPathChange() { fInPathChange = false; }
+	InPathChange() { inGuard = true; }
+	~InPathChange() { inGuard = false; }
 
 	// Delete the copy/move constructors assignment operators.
 	InPathChange(const InPathChange&) = delete;
 	InPathChange& operator=(const InPathChange&) = delete;
 	InPathChange(InPathChange&&) = delete;
 	InPathChange& operator=(InPathChange&&) = delete;
 
-	static bool inPathChange() { return fInPathChange; }
+	static bool inPathChange() { return inGuard; }
 
 private:
-	static bool fInPathChange;
+	static bool inGuard;
```

Comment:
According to our coding guidelines static members should not have leading _ (I removed f). I had to rename var since "inPathChange" conflicts with the static gettor method.